sdk/daml-assistant/integration-tests/src/DA/Daml/Assistant/QuickstartTests.hs

```diff
@@ -68,7 +68,7 @@ quickSandbox projDir = do
     withDevNull $ \devNull -> do
         callCommandSilent $ unwords ["daml", "new", projDir, "--template=quickstart-java"]
         -- TODO(#14706): remove explicit target once the default major version is 2
-        callCommandSilentIn projDir "daml build --target=2.1"
+        callCommandSilentIn projDir "daml build --target=2.2"
```

Comment:
Since this test the quickstart, should we not use the default output ? 
cc @dylant-da  ?